src/antidote_crdt_flag_ew.erl

```diff
@@ -46,38 +46,38 @@
 -define(TAG, 77).
 -define(V1_VERS, 1).
 
--export_type([flag_ew/0]).
--opaque flag_ew() :: flag_helper:flag().
+-export_type([antidote_crdt_flag_ew/0]).
```

Comment:
I think your search-replace affected too many places, the type names can be kept short ;)